ground/gcs/src/plugins/coreplugin/iboardtype.cpp

```diff
@@ -11,41 +11,76 @@ QStringList IBoardType::queryChannelBanks()
 {
     QStringList banksStringList = QStringList();
 
-    foreach(QVector<qint32> channelBank, channelBanks) {
-        qint32 tmpChannel = -1;
-        qint32 lastWrittenChannel = -1;
+    foreach(QVector<qint32> channelBank, channelBanks) 
+	{
```

Comment:
yea just to elaborate we try and mostly follow the linux kernel style convention. For function definition { start new lines. For everything else at the end of the line.